components/dashboard/src/configurations/detail/RemoveConfigurationModal.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FunctionComponent, useCallback, useState } from "react";
+import ConfirmationModal from "../../components/ConfirmationModal";
+import { useDeleteConfiguration } from "../../data/configurations/delete-configuration-mutation";
+import type { Configuration } from "@gitpod/public-api/lib/gitpod/v1/configuration_pb";
+
+type RemoveProjectModalProps = {
+    configuration: Configuration;
+    onClose: () => void;
+    onRemoved: () => void;
+};
+
+export const RemoveConfigurationModal: FunctionComponent<RemoveProjectModalProps> = ({
+    configuration,
+    onClose,
+    onRemoved,
+}) => {
+    const [disabled, setDisabled] = useState(false);
```

Comment:
We can remove this state and use the `isLoading` state from the react-query mutation, i.e. `removeConfigMutation.isLoading`